main.py

```diff
@@ -1,10 +1,25 @@
-from flask import Flask, render_template, request
+from flask import Flask, jsonify, render_template, request
+from flask_sqlalchemy import SQLAlchemy
 import json
 from flask_cors import CORS, cross_origin
 from hashlib import md5
 
-
+    
 app = Flask(__name__)
+
+#initialize configuration
+app.config["SQLALCHEMY_DATABASE_URI"] = "sqlite:///fallamoi.db"
+app.config['SQLALCHEMY_TRACK_MODIFICATIONS'] = False
+
+db = SQLAlchemy(app)
+
+#create target database model 
+class Target(db.Model):
+    id = db.Column(db.Integer, primary_key=True)
+    tid = db.Column(db.String)
+    data = db.Column(db.String,nullable=False)
```

Comment:
This should be a serializable type right ?
not a big fan on saving raw json string as string input 🤔